go-cd/README.md

```diff
@@ -0,0 +1,78 @@
+# Go CD
+
+## Building
+
+```bash
+docker build --tag appunite/go-cd:14.4.0 .
+```
+
+## Setup
+
+```bash
+docker run \
+			-v /go/etc:/etc/go \
+			-v /go/data:/data/artifacts \
+			-v /go/var:/var/go \
+			-v /go/lib:/var/lib/ldap \
+			--name go-cd-data busybox true
+```
+
+## Running
+
+```bash
+docker run -v /go/ldap:/var/lib/ldap \
```

Comment:
give a name for this docker `--name go-ci`
remove ldap from volume because it is in go-cd-data